src/Tizen.NUI/src/public/BaseComponents/ImageView.cs

```diff
@@ -885,7 +885,28 @@ public int DesiredHeight
             }
         }
 
-
+        /// <summary>
+        /// Gets or sets ReleasePolicy for image.<br />
+        /// If not supplied, the default is ReleasePolicyType.Detached.<br />
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
```

Comment:
please remove this <since_tizen> tag for hidden api. this <since_tizen> tag is attached when API is published by ACR.